ui/pages/bridge/bridge.util.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  BridgeFeatureFlagsKey,
+  BridgeFeatureFlags,
+} from '../../../app/scripts/controllers/bridge';
+import { BRIDGE_API_BASE_URL } from '../../../shared/constants/bridge';
+import { SWAPS_CLIENT_ID } from '../../../shared/constants/swaps';
+import fetchWithCache from '../../../shared/lib/fetch-with-cache';
+import { validateData } from '../../../shared/lib/swaps-utils';
+
+const CLIENT_ID_HEADER = { 'X-Client-Id': SWAPS_CLIENT_ID };
+const CACHE_REFRESH_TEN_MINUTES = 600000;
+
+// Types copied from Metabridge API
+enum BridgeFlag {
+  EXTENSION_SUPPORT = 'extension-support',
+}
+
+type FeatureFlagResponse = {
+  [BridgeFlag.EXTENSION_SUPPORT]: boolean;
+};
+// End of copied types
+
+type Validator<T> = {
+  property: keyof T;
+  type: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  validator: (value: any) => boolean;
```

Comment:
nit: value would be `unknown` here, right? This could avoid the lint line

Reply:
you're right! ty for the callout